src/components/isNotNullIsland.js

```diff
@@ -0,0 +1,8 @@
+const filter = require('through2-filter');
```

Comment:
It might be better to name this stream something like is NotBlacklisted which would be more generic and allow us to add more IDs to the list in the future. I presume null island itself would get taken care of by the check you added in the other PR about not visiting null island. If that's true, this stream might not be necessary at this time and would save a little more time during import.